model/notify_admin.go

```diff
@@ -58,9 +60,13 @@ type NotifyAdminData struct {
 	RequiredPlan    string                `json:"required_plan"`
 	RequiredFeature MattermostPaidFeature `json:"required_feature"`
 	Trial           bool                  `json:"trial"`
+	SentAt			int64				  `json:"sent_at"`
 }
 
 func (nad *NotifyAdminData) IsValid() *AppError {
+	if strings.HasPrefix(string(nad.RequiredFeature), string(WorkTemplates)) {
```

Comment:
From the front end, I am sending the requiredFeature as `worktemplate-${categoryid}`. This would give an opportunity to send notification for different templates. Hence, the prefix comparison.

![Screenshot 2023-01-07 at 16 42 36](https://user-images.githubusercontent.com/841955/211156491-d84e1c29-caf7-4fa3-949d-fa036b918aa5.png)